pkg/services/dashboards/service/dashboard_public_test.go

```diff
@@ -0,0 +1,81 @@
+//go:build integration
+// +build integration
+
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	m "github.com/grafana/grafana/pkg/services/dashboards"
```

Comment:
what does this m do?

Reply:
it's an alias so you  can use m.FunctionInDashboardsPackage instead of dashboards.FunctionInDashboardsPackage. Artifact of copy pasta